ckanext/switzerland/templates/scheming/package/snippets/additional_info.html

```diff
@@ -24,4 +24,13 @@
       </dd>
     {%- endif -%}
   {%- endfor -%}
+<dt class="dataset-label">{{ _('Metadata Download') }}</dt>
+<dd class="dataset-details">
+    <a href="/api/3/action/package_show?id={{ pkg_dict.name }}" class="btn btn-primary btn-xs" role="button" target="_blank" download="{{ pkg_dict.name }}-metadata">
+        <i class="fa fa-arrow-circle-o-down"></i> {{ _('Download') }} JSON
```

Comment:
Use "Download JSON" as translation text, as the order might change depending on the language, e.g. "JSON Herunterladen" vs. "Download JSON")